### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Erin-Cecele**

### CODE REVIEW

**Comment by Erin-Cecele** [comment]
@geealbers This isn't totally polished, but I wanted to make sure this is what you had in mind, esp. with adding more GitHub information. It feels long, which is why I added the TOC. I could use some advice on the descriptions of the various contributions and how I've outlined when they should be added to our issue tracker. It feels too redundant?

**Comment by geealbers** [comment]
@Erin-Cecele I started looking at this and found the structure of the "Identify a Contribution to Make" section a little confusing and hard to follow, and noted the repetition you mentioned as well. So just pushed a commit with a suggested restructuring. Feel free to revert and rewrite though if you don't think it's better or can further refine/improve it. I'll finishing looking at the other sections as well

**Comment by Erin-Cecele** [comment]
@geealbers Took a look at your edits to "Identify a Contribution" and that works much better. Thank you for clarifying! I'll wait to see if you have any other edits before going back in and cleaning things up.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,181 +1,163 @@
...
-
-Striving toward becoming a fully open-source project by spring 2022, we encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.
-
-There are [many ways to contribute](#identify-a-contribution-to-make) including fixing bugs, improving the documentation, and creating a new feature or theme. Please read the following guidelines carefully. Our goal is to make the process as effective and transparent as possible, and to ensure that your every contribution can become part of Quire. Thank you for taking the time.
-
-🗣**CALLING ALL FIRST-TIMERS!** — We can’t stress enough that Quire is open to contributors at all levels. That could mean making a modest fix, like correcting a typo or fixing a broken link, or more intensive contributions like adding Quire CSS styles and JavaScript, or even copyediting one of our documentation pages. Here are a few resources especially for you:
+**Thank you for your interest in contributing to Quire!** Here is the process we would like you to follow.
```
Comment (geealbers) [inline CONTRIBUTING.md:5]: > Here is the process we would like you to follow

Could we soften this a little? Just felt a little too formal and prescriptive to me. Also maybe make the link list below a numbered list so that it reads more like a "process".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,181 +1,163 @@
...
 
-Contact us at [quire@getty.edu](mailto:quire@getty.edu)
+There are two Quire repositories hosted on GitHub that can contribute to:
 
-**All are welcome.** As an open-source community, Quire is committed to providing a safe, welcoming, transparent, and inclusive environment for all our community members and those wishing to become involved. Please see our **[Code of Conduct](https://github.com/thegetty/quire/blob/main/CODE_OF_CONDUCT.md)** for more on the expectations and protections for our community members.
+- [**quire**](https://github.com/thegetty/quire):  Core Quire code including command-line interface, default starter content, and default theme.
+- [**quire-docs**](https://github.com/thegetty/quire-docs): Quire website and documentation.
 
-## Quire Code Repositories
+You are currently in the **quire-docs** repository. The Quire website and documentation are actually built using Quire, so if you haven't already, you will need to [install Quire](https://quire.getty.edu/documentation/install-uninstall/) before making any contributions.
```
Comment (geealbers) [inline CONTRIBUTING.md:24]: > The Quire website and documentation are actually built using Quire

I'd like to see this called out more. I feel like if a Quire user knows that contributing to the docs just means working on another Quire project, that's really clarifying and maybe encouraging.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,181 +1,163 @@
...
+>🗣**CALLING ALL FIRST-TIMERS!** — We can’t stress enough that Quire is open to contributors at all levels. Here are a few resources especially for you:
+>- Find [“good first issues”](https://github.com/thegetty/quire-docs/issues) tagged in our issue tracker for the thegetty/quire-docs repo
+>- Read the Quire guide to Making Your First Contribution — *in progress*
+>- Learn more about contributing to open source at [First Timers Only](http://www.firsttimersonly.com/)
 
 ## Identify a Contribution to Make
 
-**The first step to any contribution** is to post a new issue or comment on an existing issue [on our issue tracker](https://github.com/thegetty/quire/issues/). Be sure to briefly describe your proposed solution and say whether you may need help with any aspect of it. This approach has several advantages:
+Our website and docs are updated continuously for clarity and completeness but it’s not always easy to keep up with the pace of Quire’s development. We invite you to share your ideas or jump in and make edits yourself. There are several ways to contribute to the Quire website and documentation.
```
Comment (geealbers) [inline CONTRIBUTING.md:35]: > Our website and docs are updated continuously for clarity and completeness but it’s not always easy to keep up with the pace of Quire’s development. We invite you to share your ideas or jump in and make edits yourself.

I really liked this BTW

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,181 +1,163 @@
...
-Quire is very extensible. We’re eager to see more development work come from you as you create and customize your own publications in ways that can then be shared back. Quire is in the early stages of defining how this kind of contribution will work, please don’t hesitate to [email us](mailto:quire@getty.edu) with your questions or ideas.
+- Lets people know you’re working on it
+- Gives the core team and the community a chance to give feedback before you do any work
+- Helps to ensure your contribution will be accepted and successfully merged in
 
-### Write an Article
+## Start Work on Contribution
 
-We’re always interested in adding articles about specific aspects of working in Quire to our Learn knowledge base. Maybe you have some tips on modifying shortcodes or styling is CSS in the custom.css. Chances are the community would love to read about it.
+Now that you've identified your contribution, your next step will be to use GitHub and Atom to make the desired changes.
```
Comment (geealbers) [inline CONTRIBUTING.md:67]: Do we want to say "text editor" instead of Atom specifically? or "Atom or another text editor"?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,181 +1,163 @@
...
 
-[Share your article idea](https://github.com/thegetty/quire/issues/new)
+>We use GitHub to manage all our work on Quire. If you are new to GitHub, we recommend downloading [GitHub Desktop](https://desktop.github.com/). Before getting started, review the [GitHub Desktop documenation](https://docs.github.com/en/desktop) and the [Project Management with GitHub] section of the Quire documentation. An abbreviated explantion of how to submit changes via GitHub included below.
```
Comment (geealbers) [inline CONTRIBUTING.md:69]: Need a URL for [Project Management with GitHub]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,181 +1,163 @@
...
 
-The Quire documentation is hosted in a separate repository along with the website, and can be found [here](https://github.com/thegetty/quire-docs). The Quire documentation is continuously being updated for clarity and completeness, but it’s not always easy to keep up with the pace of Quire’s development!
+**Open Your Project in GitHub Desktop**
+1. At the top right, click "Current Repository" and choose the `quire-docs` repo.
 
-If there’s a section you think it missing or could be improved, we’d love for your help. And the documentation is itself a Quire site, so it should be familiar to work in.
+**Create a New Branch in GitHub Desktop**
+1. Navigate to where it says “Current Branch” at the top of your GitHub Desktop window.
+2. Click the drop-down arrow and select “new branch” to the right of the Filter box.
+3. Name your branch.
```
Comment (geealbers) [inline CONTRIBUTING.md:93]: Do we want to note any branch naming conventions here?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,181 +1,163 @@
...
+4. Select “main” where it says “Create branch based on.” This means you are creating a new branch that runs parallel to the main project.
 
-[Propose a new section or edit](https://github.com/thegetty/quire-docs/issues)<br />
-[Find an existing issue to work on](https://github.com/thegetty/quire-docs/issues)<br />
-[Read the Quire Documentation Style Guide](https://github.com/thegetty/quire-docs/wiki/Quire-Website-&-Documentation-Style-Guide)
+**Open Project File in Atom**
+1. Open Atom
+2. Click File > Open
+3. Navigate to your home directory and your select `quire-docs` folder.
+5. In Atom, expand the contents folder to see the editable markdown files and look for the file you'd like to make your changes in.
```
Comment (geealbers) [inline CONTRIBUTING.md:100]: this is numbered as "5" in the Markdown though it maybe comes out correctly as "4" in the HTML view

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,181 +1,163 @@
...
+3. Double check that your changes look right by previewing at [http://localhost:1313](http://localhost:1313).
 
-The Quire community is global and we hope to be able to continue to expand access to Quire, including to non-English speakers. Though we haven’t yet done any translation work on any of the documentation or materials, if that’s something you’re interested in doing, we’d love to talk to you.
+**Make Commits in GitHub Desktop**
+1. At the bottom left, above where it says description, write a short name for your commit. You have the option to include a description as well.
+2. Press the blue button that says “Commit to [branch name].”
 
-[Volunteer to do some translation](https://github.com/thegetty/quire-docs/issues)
+**Push Changes in GitHub**
+1. After you have made all your commits, click on the "Publish Branch" and then “Push Origin” button at the top right.
```
Comment (geealbers) [inline CONTRIBUTING.md:112]: there might also be a "Publish Repository" step in here, or it might say "Publish Repository" instead of "Publish Branch" if they cloned the project locally but have not yet pushed it to their GitHub account. Not 100% sure of this though.

Comment (geealbers) [reply CONTRIBUTING.md:112]: Actually, come to think of it, instead of cloning, I think we want them to Fork the repository which would change some things here, and in the "Submit Your Contribution" section. Do you want to try this yourself with our @quire_user account?

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,181 +1,163 @@
...
-
-A pull request doesn’t have to represent finished work. We always recommend opening a pull request early on, so others can watch or give feedback on your progress. Just mark it as a “WIP” (Work in Progress) in the subject line. As you make new commits to that branch and push them to GitHub, they’ll automatically be added to the open pull request.
+>A pull request doesn’t have to represent finished work. We always recommend opening a pull request early on, so others can watch or give feedback on your progress. When submitting a work in progress pull request, please mark it as a draft by following [these instructions](https://github.blog/2019-02-14-introducing-draft-pull-requests/). As you make new commits to that branch and push them to GitHub, they’ll automatically be added to the open pull request.
 
-This is something like what the development process may look like for you:
+**Create a Pull Request in GitHub**
```
Comment (geealbers) [inline CONTRIBUTING.md:127]: See note above, some of this might change slightly for a PR from a Fork versus a Clone

**Comment by Erin-Cecele** [comment]
@geealbers I have updated this file based on your feedback and I've included updated GH instructions based on forking the repo as @quire-user. It's ready for review/discussion.

**Review by geealbers — APPROVED** [review]
Looks great!

It _is_ a lot of steps. I'll be interested to see what kind of feedback we get from people who try to follow it. It's one of those times when I wonder if a quick video of the process would also be of use. Someone might be more likely to dive into all the steps if they first watched a short video to see that it's not all that difficult
